src/layouts/index.js

```diff
@@ -30,7 +30,7 @@ const getLanguageName = (languageCode) => {
 
 const InfoDisclaimer = () => (
   <div className="introDisclaimer">
-    <a href="/summit-2021" style={{color: '#947927'}}>Register now</a> for <strong>eBPF Summit 2021</strong> on <strong>August 18-19, 2021</strong>
+    <strong>eBPF Summit keynote and lightning talk videos are available.</strong> <a href="/summit-2021" style={{color: '#947927'}}>Watch Now!</a>
```

Comment:
```suggestion
    <strong>eBPF Summit videos are available.</strong> <a target="_blank" href="https://www.youtube.com/c/eBPFCiliumCommunity" style={{color: '#947927'}}>Watch Now!</a>
```
I don't think the videos are listed on the page yet, but we could send them straight to the YouTube channel. (Also minor wording change because we have keynotes, sessions, LTs and CTF videos this year!)

Reply:
Fyi, I pinged Phil about it so that both PDF slides + videos will be added to the conf page soon.